beep/structure.py

```diff
@@ -842,7 +842,7 @@ def __init__(self, barcode, protocol, channel_id, summary,
 
     @classmethod
     def from_raw_cycler_run(cls, raw_cycler_run, v_range=None, resolution=1000,
-                            diagnostic_resolution=500, nominal_capacity=1.1,
+                            diagnostic_resolution=1000, nominal_capacity=1.1,
```

Comment:
Since they wanted higher density for `hppc` cycles, why don't we use the finer mesh for just those instead of all diagnostic cycles?

Reply:
Sure, I can change it for just the hppc cycles. It will be more brittle though, since it will then rely on being able to locate the hppc cycle